app.rb

```diff
@@ -16,19 +16,17 @@ def list_books
       puts 'No book added yet'
       return
     end
-    @books.map do |book|
-      puts "Title: \"#{book.title}\", Author: #{book.author}"
-    end
+
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
   end
 
   def list_people
```

Comment:
@hiromataba that is because you choose y for the student if you choose n you will see it in the student list as project requirement last week
[click here](https://www.youtube.com/watch?v=vkkgrhD6aXQ)